src/s390_crypto.c

```diff
@@ -674,7 +677,8 @@ int s390_get_functionlist(libica_func_list_element *pmech_list,
 	 *   approved.
 	 * - We are in an error state.
 	 * */
-	if (((fips & ICA_FIPS_MODE) && !fips_approved(pmech_list[x].mech_mode_id))
+	if (((fips & ICA_FIPS_MODE) && !fips_approved(pmech_list[x].mech_mode_id &&
+		!fips_override(pmech_list[x].mech_mode_id)))
```

Comment:
indention?

Reply:
A general remark on indention: yes, I also see all these places where parms are not aligned in the same column. 
First observation: in Eclipse it looks different, because the tab setting is different.
Second: many function names are quite long, so there is not much space left before col 80 to place the parm lists.
What I in fact don't like is mixing of tabs and spaces. And with tabs alone you mostly cannot precisely place subsequent parm lines right below the first one. For better readability I'm then moving such parm lines to the left.
So after all I would leave all these indentations as they are, for now. Even as we have a mixture of tabs and spaces at many places in old code. But I consider to do a global rework on this at a later time.

Reply:
fine with me. I just would try to stay consistent over all files. 
I don't see any specific codding style guidelines defined for libica. For Opencryptoki we have that: https://github.com/opencryptoki/opencryptoki/blob/master/doc/coding_style.md
It seems that libica is using the linux kernel style, i.e. using tabs (8 places): https://www.kernel.org/doc/html/v4.10/process/coding-style.html
Nevertheless, define a style you want and then stick to it.